telemetry/definitions/commonDefinitions.json

```diff
@@ -3335,6 +3349,21 @@
                 }
             ]
         },
+        {
+            "name": "codeTransform_buildCompleted",
```

Comment:
same comment as above.

there is an existing `codeTransform_jobStatusChanged` event. Can "build completed" be a kind of status stored on that existing event?

I see also that gumby defined a `codeTransform_mvnBuildFailed` event. This, again, doesn't make sense. 

Instead, I suggest:

- a `codeTransform_build` event
- it has a `tool` field with values like "maven", "gradle" etc.
- its `result` field can store "success" or "failed. this is a standard field on every metric.

Reply:
Thank you for your feedback! I've addressed this in a more recent commit.